app/dossierPrograms/dossierPrograms.services.js

```diff
@@ -194,6 +217,9 @@ dossierProgramsModule.factory("dossiersProgramEventReportFactory", [
     },
 ]);
 
+var qryEventCharts =
+    dhisUrl + "eventCharts?filter=program.id\\:eq\\::programId&fields=id,displayName,displayDescription&paging=false";
```

Comment:
Probably nothing to worry about here, but as a note (I just discovered it):

> New applications and clients should avoid using the eventCharts and eventReports APIs because they are deprecated. Use the eventVisualizations API instead.

https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-238/visualizations.html#webapi_event_visualization